dev-tools/globals.ts

```diff
@@ -583,10 +584,22 @@ interface TemplateData {
 	evoLevel?: number
 	evoMove?: string
 	evos?: string[]
+	evoType?: string
```

Comment:
I would make this a string enum.

So:

`evoType?: "level" | "levelHappy" | "trade"` etc